### ISSUE TITLE
Failing `ignite new App` on external lib installation

### ISSUE THREAD

**Issue by martincik** [infinitered/ignite#241]
### Generating new app isn't working for me

The problem is with `Ignite generated code` specifically `ignite new MyApp` fails with:
```
Linking external libs./myApp/ios/myApp/Info.plist: file does not exist or is not readable or is not a regular file (Error Domain=NSCocoaErrorDomain Code=260 "The file “Info.plist” couldn’t be opened because there is no such file." UserInfo={NSFilePath=./myApp/ios/myApp/Info.plist, NSUnderlyingError=0x7fc55a400830 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}})
```

My setup:
```
npm 3.10.3
node v6.3.0
```

**Comment by GantMan** [comment]
what version of ignite do you have installed?
`ignite --version`

Also, that error looks like init failed, and was swallowed by the installer.  Can you verify you can properly build a react native project `react-native init TestForGant` without error.

**Comment by GantMan** [comment]
@martincik - any update on your setup?

**Comment by Novex** [comment]
I hit this tonight as well, first time running ignite :(

`ignite --version` reports 1.3.0 and a normal `react-native init TestForGant` generates everything properly. [Here's the full console output](https://gist.github.com/Novex/504d9fd80dc75adcd1836445ba2226dd), as well as an `ls` of the resulting directory which confirms the ios/android directories didn't get created.

Let me know if there's anything else I can run to help narrow down the problem.

Setup:
```
npm 3.7.3
node v5.9.0
```

**Comment by Novex** [comment]
Got to the bottom of it!

react-native-cli was an older 0.1.10 version, which was causing [reactNativeInit](https://github.com/infinitered/ignite/blob/master/ignite-generator/src/app/index.es#L179) to error out.

running `npm update -g react-native-cli` brought it up to 1.0.0 and fixed the problem, but I was curious why the check in `findReactNativeCli` didn't catch this. There are a few reasons - pull request incoming.

**Comment by GantMan** [comment]
awesome!

**Comment by GantMan** [comment]
thanks again @Novex  - you got a shout out in https://github.com/infinitered/ignite/releases

Closing this ticket.

**Comment by johneo** [comment]
Hi, I was running into this issue today, with node 6.2 and ignite 1.6.2. (macOS... uh... "El Capitan") I think my issue was resolved when I upgraded Xcode from 7.3.0 to 7.3.1. I tried the `ignite new` in a new terminal session as well, just to be sure.

I also noticed another potential cause of the issue. My mistake was doing `ignite new myproject` where `myproject` directory already existed and had `.git` directory and a `.gitignore` already. The same issues resulted from that. My solution was doing `ignite new aNonExistingDirectoryName` then copy the content of the result over to the existing project directory.

Hope that helps someone! (And thanks for ignite!)
